### ISSUE TITLE
Touch event support

### ISSUE THREAD

**Issue by hmdne** [opal/opal-browser#100]
While there are touch events defined, they are useless.

**Comment by hmdne** [comment]
This also involves handling classes like Touch

**Comment by jtoy** [comment]
agreed, I just spent time playing with this and looking at the source code. im not enough of a JS user to know the right way to expose this into opal.
